components/post_view/post_edited_indicator/post_edited_indicator.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import Icon from '@mattermost/compass-components/foundations/icon/Icon';
+
+interface Props {
+    postId?: string | null;
+    editedAt?: number;
+}
+
+const PostEditedIndicator = ({postId = null, editedAt = 0}: Props): JSX.Element | null => {
+    return postId && editedAt > 0 ? (
```

Comment:
Could you do an early return when either of these are falsey? It avoids some nesting, and personally I'm not a fan of multi-line ternary statements like this as well.

Reply:
yep, will do ... as well as an immediate return!

Reply:
haha ... eslint does not let me use immediate returns with ternaries 👍🏼

Reply:
I thought that actually would've gotten rid of the ternary entirely like
```jsx
if (!postId || editedAt === 0) {
    return undefined;
}

return <Stuff/>;
```

Reply:
but that is exactly the same ... so why should we use the "old-school" if clause?

Reply:
I was thinking it was going to reduce indentation, but I guess the changes I suggested didn't actually do that since you have to indent the JSX either way